app/src/main/res/layout/settings_subscribe_fragment.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
IMO, we can probably get rid of the "Subscribe" header at the top and keep the layout similar to IOS.